src/main/java/Duke.java

```diff
@@ -1,10 +1,265 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static Task[] tasks = new Task[100];
+    public static int taskCount = 0;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greet();
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        String lowerCaseUserInput;
+        boolean isLastInput = false;
+        boolean isListRequest = false;
+        boolean isMarkRequest = false;
+        boolean isUnmarkRequest = false;
+        String taskType;
+        do {
```

Comment:
Avoid long methods (>30 LoC). Consider refactoring some of the code into other classes/methods.